sdk/aqueduct/integrations/sql_integration.py

```diff
@@ -54,6 +47,37 @@ def replace_today() -> str:
     return "'" + date.today().strftime("%Y-%m-%d") + "'"
 
 
+def find_parameter_names(query: str) -> List[str]:
+    matches = re.findall(TAG_PATTERN, query)
+    return [match.strip(" {}") for match in matches]
+
+
+def find_parameter_artifacts(dag: DAG, names: List[str]) -> List[ArtifactMetadata]:
+    """
+    `find_parameter_artifacts` finds all parameter artifacts corresponding to given `names`.
+    parameters:
+        names: the list of names, repeating names are allowed.
+    returns:
+        a list of unique parameter artifacts for these names. Built-in names are omitted.
```

Comment:
Curious what python commenting style you're following? I've been trying to follow the google style guide throughout the SDK (Args, Returns, Raises): https://google.github.io/styleguide/pyguide.html#244-decision

Again this is probably something we should standardize in our design doc.

Reply:
Cool that's good to know! Let's add them to our guide doc?